autoware_adapi_v1_msgs/operation_mode/msg/OperationModeState.msg

```diff
@@ -0,0 +1,16 @@
+# constants for mode
+uint16 UNKNOWN = 0
+uint16 STOP = 1
+uint16 AUTONOMOUS = 2
+uint16 LOCAL = 3
+uint16 REMOTE = 4
+
+# variables
+builtin_interfaces/Time stamp
+uint16 mode
```

Comment:
Are there any reasons for using `uint16` instead of `uint8` here?

Reply:
No, there is no particular reason. I used uint16 for the state, so I made it the same.

Reply:
I see. In that case, I feel it's better to make it minimum, but I'll leave it to you.

Reply:
Done https://github.com/autowarefoundation/autoware_adapi_msgs/pull/17/commits/0ad9f74d323d0df7b0144f55ab042d9360088f0e.